.github/workflows/linter.yml

```diff
@@ -60,6 +60,7 @@ jobs:
           # Disable some other linters
           VALIDATE_HTML: false
           VALIDATE_NATURAL_LANGUAGE: false
+          VALIDATE_JSON: false
```

Comment:
please add a config for the json linter with specific ignore rules. We might have more json files in the future and we really want to validate them.